xml/ay_bigfile.xml

```diff
@@ -556,6 +556,13 @@ exit 0
    </para>
 
    <tip>
+    <title>jing</title>
```

Comment:
Imobach González Sosa <notifications@github.com> writes:

>> @@ -556,6 +556,13 @@ exit 0
>>     </para>
>>  
>>     <tip>
>> +    <title>jing</title>
>
> np: what about `Where to find the <literal>jing</literal> package?`

Or <package>jing</package> --which is available with DB 5 and even
GeekoDoc.

-- 
Karl Eichwalder            SUSE Linux GmbH
R&D / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg)